en/config/actuators.md

```diff
@@ -0,0 +1,116 @@
+- Tilt direction: **Towards Front** means the servo tilts towards positive X direction, whereas **Towards Right** means towards positive Y direction.
+- Minimum and maximum tilt angles: specify the physical limits of the tilt at minimum control and maximum respectively. An angle of 0 means to point upwards, then increases towards the tilt direction.
+  :::note
+  Negative angles are possible. For example tiltable multirotors have symmetrical limits and one could specify -30 as minimum and 30 degrees as maximum.
+  :::
+  :::note
+  If a motor/tilt points downwards and tilts towards the back it is logically equivalent to a motor pointing upwards and tilting towards the front.
+  :::
+- Control: depending on the airframe, tilt servos can be used to control torque on one or more axis (it's possible to only use a subset of the available tilts for a certain torque control):
+  - Yaw: the tilts are used to control yaw (generally desired).
+    If four or more motors are used, the motors can be used instead.
+  - Pitch: typically differential motor thrust is used to control pitch, but some airframes require pitch to be controlled by the tilt servos. Bicopters are among those.
+- Tiltable motors are then assigned to one of the tilt servos.
+
+![Tilt Axis](../../assets/config/tilt_axis.png)
+
+### Reversible Motors
+If reversible motor(s) are used (i.e. motor spins in direction 1 for lower output range and in direction 2 for the upper half), make sure to select the **Reversible** checkbox for those motor(s).
+Also ensure the ESC is configured appropriately (e.g. 3D mode enabled for DShot ESCs, which can be achieved via [DShot commands](../peripherals/dshot.md#commands)).
+
+
+## Actuator Outputs
+The actuators and any other output function can be assigned to any of the physical outputs.
+Each output has its own tab, e.g. the PWM MAIN or AUX output pins.
+
+PWM outputs are grouped according to the hardware groups of the autopilot.
+Each group allows to configure the PWM rate or DShot/Oneshot (if supported).
+
+:::note
+For boards with MAIN and AUX, prefer the AUX pins over the MAIN pins for motors, as they have lower latency.
+:::
+
+The AUX pins have additional configuration options for camera capture/triggering.
+Selecting these requires a reboot before they are applied.
+
+
+## Actuator Testing
+When testing actuators, make sure that:
+- Motors spin at the minimum. The sliders snap into place at the lower end: at the bottom, the motor is turned off (disarmed), whereas the next slider position commands the minimum thrust.
+  Adjust the minimum output value such that the motors spin at that slider position.
+  :::note
+  VTOL's will automatically turn off motors pointing upwards during forward flight.
+  :::
+- Servos move into the direction of the convention described above.
+  :::note
+  A trim value can be configured for control surfaces, which is also applied to the test slider.
+  :::
+
+Note the following behaviour:
```

Comment:
Should we perhaps have at least some of this stuff first as safety  warnings before the guidance? e.g. kill switch, safety button, and perhaps removing props?

Reply:
Yes we could. Although the switch in the UI already says it.